examples/cra-kitchen-sink/src/stories/index.js

```diff
@@ -60,6 +61,23 @@ const InfoButton = () => (
   </span>
 );
 
+class AsyncItemLoader extends React.Component {
+  constructor() {
+    super();
+    this.state = { items: [] };
+  }
+
+  loadItems() {
+    setTimeout(() => this.setState({ items: ['pencil', 'pen', 'eraser'] }), 1500);
+  }
+
+  render() {
+    button('Load the items', () => this.loadItems());
```

Comment:
Shouldn't this be in the constructor?

Reply:
Very clear example by the way, Thanks!

Reply:
HMR kills it when it is in the constructor. The button stays present in the knob panel but the instance of the example component is new, so the actual component tied to the knob was unmounted.

Reply:
Ok, if this is the best solution, that's fine.

Can you take a look at why the CI is disapproving of this PR?

Reply:
I restarted the build tasks